### PR TITLE
Create stack-2024-12-04.md

### PR DESCRIPTION
**Pull request by fgmccabe**
Agenda for 12/04 meeting.

### CODE REVIEW

File: stack/2024/stack-2024-12-04.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+- **Where**: zoom.us
+- **When**:  December 2nd, 17:00-18:00 UTC (December 2nd, 9am-10am Pacific Daylight Time)
+- **Location**: [Zoom call](https://zoom.us/j/91846860726?pwd=NVVNVmpvRVVFQkZTVzZ1dTFEcXgrdz09)
+
+## Participants
+
+## Agenda items
+
+1. Discuss issue #56 (Francis McCabe)
```
Comment (conrad-watt) [inline stack/2024/stack-2024-12-04.md:13]: Is this referring to https://github.com/WebAssembly/stack-switching/pull/56?

**Comment by fgmccabe** [comment]
Yes

Francis McCabe
SWE

**Comment by tlively** [comment]
I think there's some confusion here. As @bakkot called out in #1702, this is probably supposed to refer to JSPI issue 56, not stack switching issue 56. @fgmccabe, can you clarify?

(Also, @bakkot is correct that the date in the filename needs to be fixed.)

**Comment by fgmccabe** [comment]
I have amended the date.
The issue is: "should we use Promise.resolve on the return values of functions marked as Suspending?"

**Pull request merged**